neuroscout_cli/commands/run.py

```diff
@@ -25,7 +25,7 @@ def run(self):
             str(out_dir),
             'dataset',
             '--model={}'.format((bundle_path / 'model.json').absolute()),
-            '--exclude=(fmriprep.*$(?<=tsv))'.format(bundle_path.parts[-1]),
+            '--ignore=/(fmriprep.*$(?<=tsv))/'.format(bundle_path.parts[-1]),
```

Comment:
What is this `.format` doing?

Reply:
Good question, I think it's a vestige from the past, where I was ignoring other neuroscout models. Probably need to ensure its not needed anymore and remove.

Reply:
Aha, I removed it in #59 because I'm now passing in the specific bundle directory as a derivative, rather than indexing all derivs.